pkg/contracts/contracts.go

```diff
@@ -1,5 +1,14 @@
 package contracts
 
+import "os"
+
+type FileOps struct {
+	Path        string
+	MimeType    string
+	FileContent *os.File
+	VersionID   string
+}
```

Comment:
It wouldn't make sense anymore within the deploy block, because something specific to that context